test/Chainweb/Test/MultiNode.hs

```diff
@@ -300,23 +299,28 @@ replayTest loglevel v n = after AllFinish "ConsensusNetwork" $ testCaseSteps nam
     withTempRocksDb "replay-test-rocks" $ \rdb ->
     withSystemTempDirectory "replay-test-pact" $ \pactDbDir -> do
         let tastylog = step . T.unpack
+#if DEBUG_MULTINODE_TEST
```

Comment:
Do we need to keep this CPP stuff in?

Reply:
It helps to find the right place quickly in case the test needs debugging. That's how it originated. But, it probably hasn't been used in a long time. So, removing it would probably not cause any harm.